test/server/cards/01_SOR/events/RallyingCry.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Rallying Cry', function () {
+    integration(function (contextRef) {
+        describe('Rallying Cry\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['rallying-cry'],
+                        groundArena: ['death-trooper'],
+                        spaceArena: ['imperial-interceptor']
+                    },
+                    player2: {
+                        groundArena: ['rugged-survivors'],
+                        spaceArena: ['distant-patroller']
+                    }
+                });
+            });
+
+            it('should give Raid 2 to friendly units', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.rallyingCry);
+                context.player2.passAction();
+                context.player1.clickCard(context.deathTrooper);
+                expect(context.player1).toBeAbleToSelectExactly([context.p2Base, context.ruggedSurvivors]);
+                context.player1.clickCard(context.p2Base);
+                expect(context.p2Base.damage).toBe(5);
```

Comment:
Can we also confirm that imperial interceptor gets the buff by doing an attack. Other cases look great!